certora/specs/Delegation.spec

```diff
@@ -215,3 +214,33 @@ rule delegatingWithSigUpdatesVotingPower(env e, DelegationToken.Delegation deleg
         assert delegatedVotingPower(delegation.delegatee) == delegatedVotingPowerBefore + balanceOf(delegator);
     }
 }
+
+// Check that the updated voting power of a delegatee after a transfer is lesser than or equal to the total supply of tokens.
+rule updatedDelegatedVPLTEqTotalSupply(address from, address to) {
```

Comment:
```suggestion
rule updatedDelegatedVPLTEqTotalSupply(env e, address from, address to) {
```

Reply:
In this case, let's not do this change. This is because the environment `e` is ghost, and we want to use this rule as an invariant later. An invariant would not have this environment parameter, so it would make it confusing

Reply:
will be rendered moot by #108 anyway